backend/PolicyAdmin/Models/ResourcePolicy.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Altinn.Studio.PolicyAdmin.Models
+{
+    public class ResourcePolicy
+    {
+        public List<PolicyRule> Rules { get; set; }
+
+        public List<PolicyObligation> Obligations { get; set; }
+
+        public string RequiredAuthenticationLevelEndUser { get; set; }
```

Comment:
Doesn't the app policies also support a RequiredAuthenticationLevelOrg?

Reply:
Yes . fixed